.gitignore

```diff
@@ -0,0 +1,5 @@
+/.idea/
+/venv/
+*.pyc
+/llvm-project/
+/build/
```

Comment:
should there be leading `/` here? I would think those are absolute paths?

Reply:
A leading slash means we only want to ignore the `build` and `llvm-project` folder if they are in the root but I don't think it particularly matters in this instance (they are never NOT going to be in the root). I'll remove them.